### PR TITLE
(Routes component) Fix view bug in Chrome & polish

### PR DESCRIPTION
**Pull request by JulienMrgrd**
- Fix bug in Chrome (view cannot update the route button when clicked) 
- Add a darker border to dropdown
- Refactor a little bit the typescript code

### CODE REVIEW

File: src/main/webapp/app/shared/routes/route-selector.component.ts [hunk 1]
```diff
@@ -63,34 +51,24 @@ export class JhiRouteSelectorComponent implements OnInit, OnDestroy {
...
-
-    private updateRoute(updateInstance: boolean) {
+    private updateRoute() {
         this.updatingRoutes = true;
         this.routesService.findAll().subscribe((routes) => {
             this.savedRoutes = routes;
             this.routes = routes;
             this.searchedInstance = '';
 
-            if (updateInstance) {
```
Comment (deepu105) [inline src/main/webapp/app/shared/routes/route-selector.component.ts]: @JulienMrgrd why is this changed?

Comment (JulienMrgrd) [inline src/main/webapp/app/shared/routes/route-selector.component.ts]: Because we never call the updateRoute method with a false param.
